saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/node/Node.scala

```diff
@@ -79,23 +77,23 @@ class Node[T <: AnyRef](
     node
   }
 
-  def clear() = {
-    collection.clear()
-    trainingSet.clear()
-    testingSet.clear()
+  def clear(): Unit = {
+    collection.clear
+    trainingSet.clear
+    testingSet.clear
     for (e <- incoming) e.clear
```

Comment:
Good catch. Will update all usage to `clear()`. As per the Scala Style Guide, any unary- function with side-effects should have parenthesis. Will send an iteration.